docs/zh/latest/plugins/ip-restriction.md

```diff
@@ -23,22 +29,27 @@ title: ip-restriction
 
 ## 描述
 
-`ip-restriction` 可以通过以下方式限制对服务或路线的访问，将 IP 地址列入白名单或黑名单。 单个 IP 地址，多个 IP 地址 或 CIDR 范围，可以使用类似 10.10.10.0/24 的 CIDR 表示法。
+`ip-restriction` 插件可以通过将 IP 地址列入白名单或黑名单来限制对服务或路由的访问。
+
+支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（Classless InterDomain Routing，无类域间路由）范围的限制。
```

Comment:
```suggestion
支持对单个 IP 地址、多个 IP 地址和类似 `10.10.10.0/24` 的 CIDR（无类别域间路由）范围的限制。
```

Reply:
Hi, "CIDR" is not as common as "IP" or other terms. From the user's perspective and refer to the [doc style guide](https://apisix.apache.org/zh/docs/general/documentation-style-guide/), I prefer to keep the full English name.

Reply:
Hi, the full English name doesn't make sense to Chinese readers, right? 😉 Also, if readers want to use the full name, just searching the CIDR.

![image](https://user-images.githubusercontent.com/2106987/177950478-8efff267-3077-4a72-8d8e-021be8539074.png)